test/src/main/java/org/apache/accumulo/test/compaction/ExternalCompactionProgressIT.java

```diff
@@ -284,13 +284,15 @@ private static Thread getMetricsCheckerThread(AtomicLong totalEntriesRead,
           }
           TestStatsDSink.Metric metric = TestStatsDSink.parseStatsDMetric(s);
           final String metricName = metric.getName();
+          if (!metricName.startsWith("accumulo.compactor.entries")) {
```

Comment:
this was originally `startsWith(MetricsProducer.METRICS_COMPACTOR_PREFIX)` == `startsWith("accumulo.compactor.")`. Is this the same?

Reply:
Okay sounds good. I didn't look into any of the logic just noticed a diff between original and new wanted to make sure this change was intentional.